examples/mnist/notebook.jl

```diff
@@ -77,7 +77,7 @@ function MLJFlux.build(b::MyConvBuilder, rng, n_in, n_out, n_channels)
         MaxPool((2, 2)),
         Conv((k, k), c2 => c3, pad=(p, p), relu, init=init),
```

Comment:
Instead of computing the required padding to perserve image size manually above and passing it here, one can just pass the singleton `SamePad()`
https://fluxml.ai/Flux.jl/stable/models/layers/#Convolution-Models

Reply:
Good to know. I will leave it as is, for less "hidden knowledge" for begginners, but will add your alternative in the form of a comment.